Client/Impl/Commands/CreateProcessInstanceCommand.cs

```diff
@@ -58,6 +58,12 @@ public ICreateProcessInstanceCommandStep3 Variables(string variables)
             return this;
         }
 
+        public ICreateProcessInstanceCommandStep3 AddTenantId(string tenantId)
```

Comment:
❌ Please extend our current tests, so we make sure that we sending these details with the request.